src/guides/v2.4/graphql/mutations/create-requisition-list.md

```diff
@@ -0,0 +1,86 @@
+---
+group: graphql
+title: createRequisitionList mutation
+b2b_only: true
+---
+The `createRequisitionList` mutation creates a requisition list for the logged in customer.
+
+This mutation requires a valid [customer authentication token]({{page.baseurl}}/graphql/mutations/generate-customer-token.html).
+
+{:.bs-callout-info}
+Use the [`storeConfig` query]({{page.baseurl}}/graphql/queries/store-config.html) with the following attributes to determine whether requisition lists are supported:
+
+*  `btob_website_configuration_requisition_list_active`
+
+## Syntax
+
+```graphql
+mutation {
+  createRequisitionList(
+    name: String!
+    description: String
+  ) {
+    CreateRequisitionListOutput
+  }
+}
+```
+
+## Example usage
+
+The following example creates the `Frequently Ordered Products` requisition list.
+
+**Request:**
+
+``` graphql
+mutation {
+  createRequisitionList(
+    name: "Frequently Ordered Products",
+    description: "Frequently ordered products list"
+  ) {
+    list {
+      uid
+      name
+      description
+  }
+}
+```
+
+**Response:**
+
+```json
+{
+  "data": {
+    "createRequisitionList": {
+      "list": {
+          "uid": "4",
```

Comment:
Is this valid? I would expect a `uid` to contain a value like `NA==`.

Reply:
yes currently ID type is uid, this https://github.com/magento/architecture/blob/ea3e1c753b277a3f9c8e5ebf50147021b9907f22/design-documents/graph-ql/coverage/b2b/requisitionList.graphqls#L60